docs/readthedocs/usersguide.md

```diff
@@ -95,7 +105,7 @@ For a detailed description on how to do that and all the various configuration p
 ### Model and Grid Data
 
 Besides a simulation configuration, the specifications of each grid component (e.g. lines, transformers, ...) and system participant (e.g. loads, pv plants, ... ) have to be fed into the simulation.
-Within SIMONA, we use the PowerSystemDataModel (PSDM) for modeling the system components.
+Within SIMONA, we use the **PowerSystemDataModel (PSDM)** for modeling the system components.
```

Comment:
May we should include a link to the git repo of PSDM here as well?

Reply:
Good idea